docs/_policies/leavingGSB.md

```diff
@@ -42,18 +52,18 @@ To provide the best experience for current GSB researchers, the Yen systems have
 
 This data can be restored on demand, but depending on the data size, may incur an additional fee.
 
-### Exceptions
+#### Exceptions
 
 Some projects use a disproportionately large amount of data - in these instances, our team may expedite the process of purging this data in the interest of maintaining space for active GSB users.  In these situations, we will work with data owners to ensure this does not result in unacceptable data loss.
```

Comment:
In these situations, we will work with data owners to ensure this does not result in unacceptable data loss.
how about...
`In these situations, we will make our best effort to coordinate with data owners to prevent unexpected data loss.`